docs/includes/pairlists.md

```diff
@@ -74,6 +75,15 @@ be caught out buying before the pair has finished dropping in price.
 
 This filter allows freqtrade to ignore pairs until they have been listed for at least `min_days_listed` days.
 
+#### PerformanceFilter
+
+Sorts pairs by past trade performance, as follows:
+1. Positive performance.
+2. No closed trades yet.
+3. Negative performance.
+
+Trade count is used as a tie breaker.
```

Comment:
I acknowledge that alphabetical sort as a secondary tie breaker is omitted. But with that being so unlikely (and presumably inconsequential), it seems fine to omit.